### ISSUE TITLE
Svelte 4 warning during build

### ISSUE THREAD

**Issue by punyflash** [inertiajs/inertia#1627]
### Version:

- `@inertiajs/svelte` version: 1.0.9
- `svelte` version: 4

### Describe the problem:

```sh
[vite-plugin-svelte] /node_modules/.pnpm/@inertiajs+svelte@1.0.9_svelte@4.1.1/node_modules/@inertiajs/svelte/src/Link.svelte:25:0 A11y: <svelte:element> with click, dblclick, mousedown, mousemove, mouseout, mouseover, mouseup handlers must have an ARIA role
23: </script>
24: 
25: <svelte:element
    ^
26:   this={as}
27:   use:inertia={{
```
### Steps to reproduce:

```sh
vite build
```

**Comment by reinink** [comment]
Hey can you provide a minimal reproduction of this as a git repo? This makes it much easier for us to reproduce things on our end and ultimately fix it if there is an issue.

Also, for what it's worth our Svelte demo app uses version 4, and I don't see this issue there. You can find that demo app here:

https://github.com/inertiajs/inertia/tree/master/playgrounds/svelte

**Comment by punyflash** [comment]
> Hey can you provide a minimal reproduction of this as a git repo? This makes it much easier for us to reproduce things on our end and ultimately fix it if there is an issue.
> 
> Also, for what it's worth our Svelte demo app uses version 4, and I don't see this issue there. You can find that demo app here:
> 
> https://github.com/inertiajs/inertia/tree/master/playgrounds/svelte


This issue is quite small, I don't feel a need to create whole repository to show it. It occurred in case when `Link.svelte` is used. This is just Svelte's A11y warning

**Comment by jdion84** [comment]
@punyflash  did you ever figure this out?

**Comment by punyflash** [comment]
> @punyflash did you ever figure this out?
Fix in #1635. Also, I'd rather not recommend you to use inertia with Svelte 4, as #1621 is even worse

**Comment by ingLomeland** [comment]
anyone figured out this problem? I've got a fresh project and not even using the Link component, but I still get this warning

**Comment by driesvints** [comment]
A PR was sent in but was stalled. It's best that someone picks up the work in a new PR. Thanks

**Comment by reinink** [comment]
Decided to fix this using a `svelte-ignore` (see #1858), and that has been released as part of [v1.0.16](https://github.com/inertiajs/inertia/releases/tag/v1.0.16) 👍
